src/main/java/org/isf/operation/model/Operation.java

```diff
@@ -48,39 +51,33 @@
 public class Operation extends Auditable<String> {
 
 	@Id
-	@Column(name="OPE_ID_A")	    
-    private String code;
+	@Column(name = "OPE_ID_A")
+	private String code;
 
 	@NotNull
-	@Column(name="OPE_DESC")
-    private String description;
+	@Column(name = "OPE_DESC")
+	private String description;
 
 	@NotNull
 	@ManyToOne
-	@JoinColumn(name="OPE_OCL_ID_A")
-    private OperationType type;
+	@JoinColumn(name = "OPE_OCL_ID_A")
+	private OperationType type;
 
 	@NotNull
-	@Column(name="OPE_STAT")
-    private Integer major;
-
-	/*
-	 * //TODO: replace "integers" values with mnemonic ones, CHAR(1) -> VARCHAR(10)
-	 * "1" = OPD / ADMISSION
-	 * "2" = ADMISSION
-	 * "3" = OPD
-	 */
-	@Column(name="OPE_FOR")
-    private String operFor;
-	
-	
+	@Column(name = "OPE_STAT")
+	private Integer major;
+
+	@Enumerated(EnumType.STRING)
+	@Column(name = "OPE_FOR", columnDefinition = "ENUM('opd', 'admission', 'opd_admission')")
```

Comment:
the order matters, please use this:
```suggestion
	@Column(name = "OPE_FOR", columnDefinition = "ENUM('opd_admission', 'admission', 'opd')")
```